src/lib/components/LoginModal.svelte

```diff
@@ -25,11 +25,14 @@
 				v{PUBLIC_VERSION}
 			</div>
 		</h2>
-		<p class="px-4 text-lg font-semibold leading-snug text-gray-800 sm:px-12">
-			This application is for demonstration purposes only.
+		<p
+			class="px-4 text-lg font-semibold leading-snug text-gray-800 sm:px-12"
+			style="text-wrap: balance;"
+		>
+			Please Sign in with Hugging Face to continue
```

Comment:
This text shows on the modal even when login is disabled

![image](https://github.com/huggingface/chat-ui/assets/25119303/2a709474-8484-467e-84db-5b7a8848ca1e)